.code-samples.meilisearch.yaml

```diff
@@ -162,15 +162,128 @@ get_all_tasks_1: |-
             print(error)
         }
     }
-get_keys_1: |-
-    client.keys { (result: Result<Key, Swift.Error>) in
-        switch result {
-        case .success(let key):
-            print(key)
-        case .failure(let error):
-            print(error)
-        }
+get_on_key_1: |-
+  self.client.getKey(key: d0552b41536279a0ad88bd595327b96f01176a60c2243e906c52ac02375f9bc4) { result in
```

Comment:
You don't need to use the types anymore `(result: Result<Key, Swift.Error>)` ? ⚡